app/views/developers/_availability.html.erb

```diff
@@ -0,0 +1,5 @@
+<span class="text-gray-900 text-sm <%= "font-medium" if developer.available_now? %>">
```

Comment:
This file can be removed, right?

Reply:
Good spot! I must have forgotten to delete this file when cherry-picking the component commit. I guess this is why we code review 😉